contracts/modules/capital/Ramm.sol

```diff
@@ -686,20 +685,21 @@ contract Ramm is IRamm, MasterAwareV2, ReentrancyGuard {
     uint spotPriceA = 1 ether * state.eth / state.nxmA;
     uint spotPriceB = 1 ether * state.eth / state.nxmB;
 
-    uint priceA;
-    uint priceB;
-
     // underflow is desired
     unchecked {
       uint averagePriceA = uint(currentObservation.priceCumulativeAbove - firstObservation.priceCumulativeAbove) * 1e9 / elapsed;
       uint averagePriceB = uint(currentObservation.priceCumulativeBelow - firstObservation.priceCumulativeBelow) * 1e9 / elapsed;
 
       // keeping min/max inside unchecked scope to avoid stack too deep error
-      priceA = Math.min(averagePriceA, spotPriceA);
-      priceB = Math.max(averagePriceB, spotPriceB);
+      uint priceA = Math.min(averagePriceA, spotPriceA);
+      uint priceB = Math.max(averagePriceB, spotPriceB);
+      internalPrice =  priceA + priceB - 1 ether * capital / supply;
     }
 
-    return priceA + priceB - 1 ether * capital / supply;
+    uint maxPrice = 3 * 1 ether * capital / supply; // 300% BV
+    uint minPrice = 35 * 1 ether * capital / supply / 100; // 35% BV
```

Comment:
should we add global constants for the limits? i.e. 100% and 35%?

Reply:
We can, but this is the only place where it is used, and it would just add one more division for max since it would need to be divided by 100 if we set the limit to 300